docs/userguide.md

```diff
@@ -387,3 +387,164 @@ employed by deployment:
 ```bash
 NAMESPACED=y FEDERATION_NAMESPACE=<namespace> ./scripts/delete-federation.sh
 ```
+
+## Higher order behaviour
```

Comment:
How about `Advanced Features`

Reply:
The current title seems apt to me, given that this section intends to provide details about the higher level API and associated controllers.